zerver/webhooks/grafana/view.py

```diff
@@ -0,0 +1,46 @@
+from typing import Any, Dict
+
+from django.http import HttpRequest, HttpResponse
+
+from zerver.decorator import api_key_only_webhook_view
+from zerver.lib.request import REQ, has_request_variables
+from zerver.lib.response import json_success
+from zerver.lib.webhooks.common import check_send_webhook_message
+from zerver.models import UserProfile
+
+NOTIFICATIONS_STATE_TYPE = ['paused', 'alerting', 'pending', 'no_data']
+
+GRAFANA_TOPIC_TEMPLATE = '{title}: {state}'
+GRAFANA_MESSAGE_TEMPLATE = '{ruleName}: {message}. Please check out the details' + \
```

Comment:
Maybe just make `{ruleName}` link to the `{ruleUrl}` to reduce the verbosity. And then add the `{detail}` after a newline.